features/layout-direction-override.yml

```diff
@@ -0,0 +1,17 @@
+name: Layout direction override
+description: The `unicode-bidi` and `direction` CSS properties override the Unicode layout algorithm. For HTML documents, authors should use the `dir` global HTML attribute and `<bdo>` HTML element instead.
```

Comment:
As written, this sounds like an anti-feature. Is it possible you could link to some history about this feature? I'm wondering who it's _for_, if it's not deprecated.

Reply:
Good point- I'm not seeing historical posts about this, but both [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi) and [WebReference](https://webreference.com/css/properties/unicode-bidi/) say it is explicitly for DTD designers, so I updated the description.